tools/monorepo-utils/src/core/github/repo.ts

```diff
@@ -9,8 +9,8 @@ import { Repository } from '@octokit/graphql-schema';
 import { graphqlWithAuth, octokitWithAuth } from './api';
 
 export const getLatestGithubReleaseVersion = async ( options: {
-	owner?: string;
-	name?: string;
+	owner: string;
```

Comment:
Checked all of these, callers are never null or undefined and these shouldn't be undefined either because the requests won't work at all without them.